### PR TITLE
Upgrade hyperf to v2.2

### PR DESCRIPTION
**Pull request by limingxinleo**

### CODE REVIEW

**Comment by waghanza** [comment]
will it work ? I mean since`hyperf` **2.2** is in `rc` status 

does **2.2** support only `php8` or `php>=7.4` ?

**Comment by waghanza** [comment]
composer install the beta https://the-benchmarker.semaphoreci.com/jobs/59cd62da-de6c-41c2-8251-7bda12706739#L2123 @limingxinleo 

according to https://getcomposer.org/doc/articles/versions.md it should take `rc4`  available in https://packagist.org/packages/hyperf/hyperf

File: php/hyperf/composer.json [hunk 1]
```diff
@@ -7,13 +7,19 @@
...
   "autoload": {
     "psr-4": {
       "App\\": "app/"
     }
+  },
+  "minimum-stability": "dev",
+  "prefer-stable": true,
+  "config": {
+    "optimize-autoloader": true,
+    "sort-packages": true
```
Comment (waghanza) [inline php/hyperf/composer.json:23]: no need, done in the dockerfile https://github.com/the-benchmarker/web-frameworks/blob/e685546fb7cae7d926753bbd067c6cb66c1f3ebb/php/Dockerfile#L39

**Comment by limingxinleo** [comment]
`hyperf 2.2` support 7.4 and 8.0

`2.2` will be released in recent days and then I fix it

**Review by waghanza — CHANGES_REQUESTED** [review]
last thing, could you pin this version in `config.yaml` ?

**Comment by limingxinleo** [comment]
@waghanza of course

**Review by waghanza — APPROVED** [review]

**Pull request merged**